proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -320,6 +336,10 @@ message Message {
   MessageOptions message_options = 10;
   // implicitly_deprecated is true if its enclosing file or parent element is deprecated.
   bool implicitly_deprecated = 11;
+  // All options that are present on the message. This is a super-set of
+  // message_options and uses a dynamic representation so it can also
+  // accommodate custom options with arbitrary types.
+  repeated FieldLiteral options = 12;
```

Comment:
It's weird that we have `message_options` and then `options` where `options` is defined as "all options on the message". Do you instead want to deprecate `message_options` and say that this supersedes it:?

Reply:
Yea we can do that. Should we also do the same with `enum_options` , `method_options` and 'service_options` as well?

Reply:
Yes. In all cases, the new `repeated FieldLiteral options` field deprecates the old `*_options` field.

Reply:
Perfect

Reply:
Added `[deprecated = true]` to all `*_options` fields

Reply:
I can completely remove `*_options` fields from the messages in a follow up PR after the changes in core ships.